notebooks/official/model_monitoring/get_started_with_model_monitoring_xgboost.ipynb

```diff
@@ -133,186 +138,181 @@
     {
       "cell_type": "markdown",
       "metadata": {
-        "id": "f3848df1e5b0"
+        "id": "f0316df526f8"
       },
       "source": [
-        "## Installation\n",
-        "\n",
-        "Install the packages required for executing this notebook."
+        "## Get started"
       ]
     },
     {
-      "cell_type": "code",
-      "execution_count": null,
+      "cell_type": "markdown",
       "metadata": {
-        "id": "b24b232ee039"
+        "id": "de998a3953c9"
       },
-      "outputs": [],
       "source": [
-        "# Install required packages.\n",
-        "! pip3 install --quiet --upgrade google-cloud-aiplatform \\\n",
-        "                                 google-cloud-bigquery"
+        "### Objective\n",
+        "\n",
+        "In this notebook, you learn to use the `Vertex AI Model Monitoring` service to detect feature skew and drift in the input predict requests for XGBoost models.\n",
```

Comment:
remove back ticks

- Vertex AI Model Monitoring.  (It's a product/service name)